network_monitor.py

```
@@ -0,0 +1,299 @@
+                            pass
+                        return True
+                except:
+                    continue
+            
+            return False
+        except Exception:
+            return False
+
+def generate_random_name(length=8):
+    """生成随机名称"""
+    return ''.join(random.choices(string.ascii_lowercase + string.digits, k=length))
```

Ruff (0.12.2)

Undefined name `string`

(F821)